res/res_pjsip_refer.c

```diff
@@ -733,6 +736,49 @@ static void refer_blind_callback(struct ast_channel *chan, struct transfer_chann
 
 	pbx_builtin_setvar_helper(chan, "SIPREFERRINGCONTEXT", S_OR(refer->context, NULL));
 
+	get_xfrdata = pbx_builtin_getvar_helper(chan, "GET_TRANSFERRER_DATA");
```

Comment:
get_xfrdata is only valid for as long as the channel lock is held, if it's not held then there's no guarantee, for use outside it needs to be copied using ast_strdupa

Reply:
Thanks, I didnt know it.